west.yml

```diff
@@ -157,11 +157,8 @@ manifest:
     # Other third-party repositories.
     - name: cmock
       path: test/cmock
-      revision: 9d092898ef26ece140d9225e037274b64d4f851e
-      remote: throwtheswitch
-    - name: unity
-      path: test/cmock/vendor/unity
-      revision: 74cde089e65c3435ce9aa87d5c69f4f16b7f6ade
+      submodules: true
+      revision: pull/401/head
```

Comment:
@nordic-krch can you please set a SHA here?

Reply:
updated to SHA (latest master).